app/grandchallenge/evaluation/templates/evaluation/evaluation_detail.html

```diff
@@ -205,6 +205,21 @@ <h3 class="card-title">Contact User</h3>
 
             {% include "evaluation/evaluation_incomplete_jobs_detail.html" %}
 
+            {% if object.status != object.SUCCESS or incomplete_jobs %}
+                <div class="card-body">
+                    <h3 class="card-title">Error Message</h3>
+
+                    {% if not incomplete_jobs %}
+                        All prerequisite jobs succeeded.
+                        {% if object.error_message %}
+                            Evaluation failed due to the following error: {{ object.error_message }}
+                        {% endif %}
+                    {% else %}
+                        Some jobs are incomplete. Check the status and error messages of the prerequisite jobs above.
+                    {% endif %}
```

Comment:
Unnecessary negation of boolean test.

```suggestion
                    {% if incomplete_jobs %}
                        Some jobs are incomplete. Check the status and error messages of the prerequisite jobs above.
                    {% else %}
                        All prerequisite jobs succeeded.
                        {% if object.error_message %}
                            Evaluation failed due to the following error: {{ object.error_message }}
                        {% endif %}
                    {% endif %}
```